test/replaceAll.js

```diff
@@ -47,4 +42,66 @@ describe('replaceAll', function() {
     assert.strictEqual(RA.replaceAll('a', 'c')('aba'), 'cbc');
     assert.strictEqual(RA.replaceAll('a')('c')('aba'), 'cbc');
   });
+
+  context('given searchValue is a non-global RegExp', function() {
+    specify('should throw Error', function() {
+      assert.throws(() => RA.replaceAll(/a/, 'c', 'abc'));
+    });
+  });
+});
+
+describe('replaceAllPolyfill', function() {
```

Comment:
We're missing currying test for polyfill which should be the same as this one: https://github.com/char0n/ramda-adjunct/blob/f4cca77cde95afdeafbf6c5604731a19bc99a2f7/test/replaceAll.js#L39